src/registrar/tests/test_models.py

```diff
@@ -1654,11 +1669,28 @@ def test_create_or_update_organization_type_existing_instance_updates_election_b
         self.assertEqual(domain_information.is_election_board, False)
         self.assertEqual(domain_information.organization_type, DomainRequest.OrgChoicesElectionOffice.CITY)
 
-        # Try reverting setting an invalid value for election board (should revert to False)
+    @less_console_noise_decorator
+    def test_create_or_update_organization_type_existing_instance_updates_election_board_to_none(self):
```

Comment:
These test names are very long, we may be competing with Java here. Can we shorten the name and add more into the comment. Great unit test comments follow the given/when/then story concept and while we don't enforce that as a must, those main points should ideally be covered in the comment if it's not immediately apparent. Here I don't think I would know exactly how election board would get updated to none using our system, if I hadn't seen the other comment you made.

Reply:
Done. I also changed the initial is_election_office value to false, since that more accurately describes the situation where a request was only completed to the point of generic_org_type.

Reply:
I made a few changes in addition to this after realizing that setting the initial value on creating a domain information object to False made it fail validation.

Reply:
@zandercymatics I accidentally typed False when I meant None here, I have updated the comment. Previously the assumption in validation was that is_election_board would always be a boolean value, hence is_election_type was compared directly to is_election_board. When the latter is None, that causes the check to always fail even though there are now valid states where is_election_board=None (even when the generic org type can technically have an election office).